popup.js

```diff
@@ -1,3 +1,5 @@
+// @ts-nocheck
```

Comment:
Why we need this line because we only have `.js.` ?

Reply:
It's default one also. just leave it in. I use this because browser can't recognize the `chrome` variable. because the `chrome` variable comes from the browser and have to type define for it. That's it.